sources/core/Xenko.Core.Mathematics/Plane.cs

```diff
@@ -426,6 +426,24 @@ public static Vector3 Project(Plane plane, Vector3 point)
             return result;
         }
 
+        /// <summary>
+        /// Creates a plane of unit length.
+        /// </summary>
+        /// <param name="normalX">The X component of the normal.</param>
+        /// <param name="normalY">The Y component of the normal.</param>
+        /// <param name="normalZ">The Z component of the normal.</param>
+        /// <param name="planeD">The distance of the plane along its normal from the origin.</param>
+        /// <param name="result">When the method completes, contains the normalized plane.</param>
+        public static void Normalize(float normalX, float normalY, float normalZ, float planeD, out Plane result)
+        {
+            float magnitude = 1.0f / (float)(Math.Sqrt((normalX * normalX) + (normalY * normalY) + (normalZ * normalZ)));
+
+            result.Normal.X = normalX * magnitude;
```

Comment:
I don't know if all compilers accept it, my one does since it can detect all fields being set so the constructor isn't needed. I do know we already do it in a few places, Vector3 does this in many of its code, eg.
https://github.com/stride3d/xenko/blob/717771d680686003563671281f4a817b629e7d9d/sources/core/Xenko.Core.Mathematics/Vector3.cs#L803-L808

Reply:
OK. It is probably fine then.